core/common/src/main/java/alluxio/PropertyKey.java

```diff
@@ -33,11 +36,92 @@
  */
 @ThreadSafe
 public final class PropertyKey implements Comparable<PropertyKey> {
+
+  /**
+   * The level consistency check should apply to a certain property key.
+   */
+  public enum ConsistencyCheckLevel {
```

Comment:
move the inner classes lower - fields are more interesting than inner classes, so they should go higher up.

Reply:
Done.